### PR TITLE
Rewrite attendee guide

### PR DESCRIPTION
**Pull request by akhmerov**
Still TODO: we need a second volunteer monitoring conduct. @giulia-pacchioni you up for it?

### CODE REVIEW

**Comment by giulia-pacchioni** [comment]
> Still TODO: we need a second volunteer monitoring conduct. @giulia-pacchioni you up for it?

Sure! This is about monitoring the email address attendees can use to report misconduct, right?

**Comment by akhmerov** [comment]
Correct.

File: attendeeguide.md [hunk 1]
```diff
@@ -1,32 +1,22 @@
...
+* Follow the moderator instructions.
 * Use your real name when logging in.
-* Check your internet connection and microphone sound quality in advance.
-* Coordinate with colleagues to watch jointly.
-* Consider keeping the camera running as a courtesy to the speaker.
-* Use the chat to contact the moderators if you have an important technical remarks.
-
-## Asking Questions
+* Be professional, polite and respectful.
+* Click the "raise hand" button at the participants pane if you want to ask a question. The moderator will announce and unmute you.
```
Comment (akhmerov) [inline attendeeguide.md:8]: Add a screenshot.

File: attendeeguide.md [hunk 2]
```diff
@@ -1,32 +1,22 @@
...
-You cannot unmute your microphone on your own. Zoom has a “raise hand” feature which you should use to alert the moderator about a question:
-
-1. Open the participants pane (we recommend doing this at the beginning of the talk).
-2. Click the “raise your hand” button. Your name, now with a raised hand symbol, will go to the top of the participants list, which the moderator will be monitoring.
-3. The moderator will alert the speaker, announce your name, and unmute your microphone.
-4. Ask your question!
-5. When the interaction ends, the moderator will mute your microphone and “lower your hand”.
+* Try to stay for the complete event.
+* Please check your internet connection and microphone sound quality in advance if you want to ask questions.
+* Consider keeping the camera running as a courtesy to the speaker.
```
Comment (akhmerov) [inline attendeeguide.md:15]: Remove this point: many streams seem to cause problems with a CPU.

Can add an avatar instead.

**Pull request merged**